source/_layouts/post.twig

```diff
@@ -18,3 +18,11 @@
         </div>
     {% endfor %}
 {% endblock %}
+
+{% block description_block %}
+    {%- if page.description -%}
+        {{ page.description |replace("\n", " ") }}
```

Comment:
Crap my bad, this line was wrong (see fix e9db1fd930e22fca851f1f7a328fec6c9b1b8bc4) and it broke a few posts.